osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -99,16 +99,18 @@ protected virtual Triangle CreateTriangle()
 
             const float size = 100;
 
-            return new Triangle
+            return new EquilateralTriangle
             {
                 Origin = Anchor.TopCentre,
+

```

Comment:
Wouldn't say this is inconsistent at all in that case. Just that there's no guideline or style for this.

Can put a discussion up, sure, but I'm not changing this right now unless it's enforced or there's already currently a guideline - I see it as a waste of time. Especially considering taiko is coded strictly this way.